graphbolt/CMakeLists.txt

```diff
@@ -58,6 +58,19 @@ if(USE_CUDA)
   if(DEFINED ENV{CUDAARCHS})
     set(CMAKE_CUDA_ARCHITECTURES $ENV{CUDAARCHS})
   endif()
+  set(CMAKE_CUDA_ARCHITECTURES_UNFILTERED ${CMAKE_CUDA_ARCHITECTURES})
```

Comment:
What is the version of the latest unsupported GPUs? we should doc about in which case we can clean up.

Reply:
The comments are right below, Volta+ (e.g. V100) For windows and Pascal+(e.g. P100) for Linux.

Reply:
Our CI supports the new code.

Reply:
And users can build from source for older architectures. I wonder if we can ship 2 binaries for graphbolt, one for older architectures and one for newer?

Reply:
https://github.com/dmlc/dgl/pull/7249, this PR has the modification to the CMake file seperately. May want to move the discussion there.

Reply:
I have found a workaround without dropping support for old architectures. However, the CI is not happy while it works on my local environment.